docs/source/overview.rst

```diff
@@ -4,14 +4,34 @@
 Overview
 ########
 
+``mirdata`` is a library which aims to standardize how audio datasets are accessed in Python, removing the need for writing custom loaders in every project.
+Working with datasets usually requires an often cumbersome step of downloading data and writing load functions that load e.g. matching audio and
+annotation files into some standard format to be used for experimenting or evaluating. ``mirdata`` does all of this for you:
 
-``mirdata`` works based on two main components: ``datasets loaders`` and ``indexes``. In general terms, a ``dataset loader`` contains customized code for working with the different elements of a dataset (audio, annotations, metadata); and an ``index`` has the information about the folder structure the dataset should have and the checksums of its different elements, and is used to load and validate them.
+.. code-block:: Python
 
+    import mirdata
 
-All ``datasets loaders`` have the following functionalities: ``readme()``, ``cite()``, ``download()``, ``validate()``. Besides these common functionalities, each ``dataset loader`` has its own functions and attributes depending on the nature of the dataset. For example, most datasets consist of a collection
-of ``tracks``, then most ``dataset loaders`` will have a ``Track`` attribute. Moreover, depending on the type of ``annotation`` the dataset has, the track will have different
-attributes such as ``beats`` or ``chords``. When the annotations are ``time-series``, they have their own ``mirdata`` ``data-type``. If the annotations are static over the whole track, they are included as ``metadata``. See the :ref:`tutorial` for a detail explanation on how to interact with the library.
+    tinysol = mirdata.initialize('tinysol')
+    tinysol.download()
 
+    # get annotations and audio for a random track
+    example_track = tinysol.choice_track()
+    instrument = example_track.instrument_full
+    pitch = example_track.pitch
+    y, sr = example_track.audio
+
+``mirdata`` loaders contain methods to:
+
+- ``download()``: download (or give instructions to download) a dataset
+- ``load_*()``: load a dataset's files (audio, metadata, annotations, etc.) into standard formats 
```

Comment:
its in the datasets too! I put this bullet here because i noticed some people using mirdata to download but still writing their own load functions. Even though we expose the loaded data directly through the attributes I wanted to make it super clear that we wrote these functions for people

Reply:
True! You can directly use the annotations path, forgot :) That should be in the `work datasets remotely` section of the tutorial IMO @genisplaja

Reply:
Yes! There is a section called `Accessing annotations remotely` where there's a code example of that. I can give it another title if you want!

Reply:
I agree it's more about remote access, but I wanted something up-front in the dataset description so that people don't miss the point that we're providing functions for actually loading the annotations files. I realize it's not the main use case, and I agree it's good to talk about it in more detail as a "using remote data" feature. I don't have another idea for how to clearly say "we have functions for loading specific annotations you don't need to write them yourself" - do you have an idea for how?

Reply:
What about something like that:
```
``load_*()``: load dataset's files (audio, metadata, annotations, etc.) into standard formats so additional functions are not needed
```

Reply:
@magdalenafuentes I'll update with Pedro's suggestion and merge, and we can adjust further as needed!